### PR TITLE
feat: add shutter child lock support

### PR DESCRIPTION
**Pull request by YogevBokobza**
## Description

> Added support for controlling child lock button on Switcher Runners

**Related issue (if any):** fixes #issue_number_goes_here

## Checklist

- [x] I have followed this repository's contributing guidelines.
- [x] I will adhere to the project's code of conduct.

## Additional information

> Anything else?

### CODE REVIEW

**Comment by thecode** [comment]
Can you separate the docs fixes from this PR? it looks strange to fix docs related from previous PRs together with a feature PR

**Comment by YogevBokobza** [comment]
@thecode Don't know if you missed this PR or not.. please need your attention. whenever you can.
Thanks.

File: src/aioswitcher/device/__init__.py [hunk 1]
```diff
@@ -268,6 +268,31 @@ def value(self) -> str:
...
+    def __new__(cls, value: str, display: str) -> "ShutterChildLock":
+        """Override the default enum constructor and include extra properties."""
+        new_enum = object.__new__(cls)
+        new_enum._value = value  # type: ignore
+        new_enum._display = display  # type: ignore
+        return new_enum
+
+    @property
+    def display(self) -> str:
+        """Return the display name of the swing."""
```
Comment (thecode) [inline src/aioswitcher/device/__init__.py]:
```suggestion
        """Return the display name of the shutter child lock."""
```

File: src/aioswitcher/device/__init__.py [hunk 2]
```diff
@@ -268,6 +268,31 @@ def value(self) -> str:
...
+        return self._display  # type: ignore
+
+    @property
+    def value(self) -> str:
+        """Return the value of the swing."""
```
Comment (thecode) [inline src/aioswitcher/device/__init__.py]:
```suggestion
        """Return the value of the shutter child lock."""
```

**Review by thecode — CHANGES_REQUESTED** [review]

**Review by thecode — APPROVED** [review]

**Comment by YogevBokobza** [comment]
@TomerFi  Can you make a release, please?
I guess a minor is ok.
Thanks!

**Comment by TomerFi** [comment]
Done: [5.1.0](https://github.com/TomerFi/aioswitcher/releases/tag/5.1.0).

**Pull request merged**